internal/csv/csv.go

```diff
@@ -10,7 +10,16 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-const InfrastructureFeaturesAnnotation = "operators.openshift.io/infrastructure-features"
+const (
+	InfrastructureFeaturesAnnotation = "operators.openshift.io/infrastructure-features"
+	DisconnectedAnnotation           = "features.operators.openshift.io/disconnected"
+	FipsCompliantAnnotation          = "features.operators.openshift.io/fips-compliant"
```

Comment:
NIT, don't yell at me! :P

```suggestion
	FIPSCompliantAnnotation          = "features.operators.openshift.io/fips-compliant"
```

Reply:
Not a nit at all to me. +1